index.bs

```diff
@@ -347,6 +347,8 @@ connection IDs are chosen, agents are restricted from changing IP or port
 without establishing a new QUIC connection.  In such cases, agents must
 establish a new QUIC connection in order to change IP or port.
 
+The ALPN used must be "osp".
```

Comment:
More context here would be nice, like a spec link for ALPN. And shouldn't this be in the TLS 1.3 section, since ALPN is part of the handshaking stuff?

Reply:
Done

Reply:
Done